apps/storybook/src/Toolbar.stories.tsx

```diff
@@ -92,6 +98,16 @@ export const Default = Template.bind({});
 export const Responsive = Template.bind({});
 Responsive.args = { narrow: true };
 
+export const DocumentInteractions = Template.bind({});
+DocumentInteractions.args = {
+  narrow: true,
+  interactions: [
+    { shortcut: 'Wheel', description: 'Turn' },
+    { shortcut: 'Space', description: 'Accelerate' },
+    { shortcut: 'Ctrl+Alt', description: 'Do a backflip' },
```

Comment:
I didn't mind your previous syntax: `{ 'Wheel': 'Turn', ... }`, I thought it was more concise, but I don't have strong feelings either way.

Reply:
I find that sometimes `Record` can be a bit awkward to go through and understand. 

While this is longer, I find it more explicit and it does not need `Object.values/entries` fiddling